src/app/cards/munchkin/cards/treasures/permanent-item/model/permanent-item-card.ts

```diff
@@ -0,0 +1,24 @@
+
+import { PermanentItemType } from './permanent-item-type';
+import { ImageBottomLayout } from 'src/app/cards/infrastructure/cards';
+import { TreasureCard } from '../../treasure-card';
+
+/**
+ * Represents a permanent item card. Extends [[DoorCard]].
+ */
+export class PermanentItemCard extends TreasureCard {
+
+  bonus: string;
+  specificity: string;
+  kind: string;
+  isHuge: boolean;
+
+  constructor(o: any = null) {
+    super(PermanentItemType, o);
+    this.bonus = o?.bonus || '1';
+    this.specificity = o?.specificity || '';
+    this.kind = o?.kind || '';
+    this.isHuge = o?.isHuge || false;
+    this.layout = o?.layout || ImageBottomLayout;
```

Comment:
We could possibly inject a default configuration on a per-card type basis, so we don't have to change these in each component separately.